VultisigApp/VultisigApp/Tss/TssMessenger.swift

```
@@ -70,15 +77,22 @@ final class TssMessengerImpl: NSObject, TssMessengerProtocol {
             }
             req.setValue("Basic \(authentication.base64EncodedString())", forHTTPHeaderField: "Authorization")
         }
-        
-        guard let encrypedBody = body.aesEncrypt(key: self.encryptionKeyHex) else {
+        var encryptedBody: String? = nil
+        if self.encryptGCM {
+            print("decrypt with AES+GCM")
+            encryptedBody = body.aesEncryptGCM(key: self.encryptionKeyHex)
```

lets make it throws instead of returning nil? 
To not make nil checks after